kurtosis-module/impl/module_io/params_deserializer.go

```diff
@@ -58,6 +55,20 @@ func DeserializeAndValidateParams(paramsStr string) (*ExecuteParams, error) {
 			paramsObj.Participants[idx].ELClientImage = defaultElClientImage
 		}
 
+		if participant.ELClientLogLevel == unspecifiedLogLevel {
+			var elClientParticipanLogLevel participant_log_level.ParticipantLogLevel
+			if paramsObj.ClientLogLevel != unspecifiedGlobalParticipanLogLevel {
+				elClientParticipanLogLevel = paramsObj.ClientLogLevel
+			} else {
+				elClientParticipanLogLevel = defaultELClientParticipantLogLevel
+			}
+			elClientLogLevel, found := elClientsParticipantLogLevels[elClientType][elClientParticipanLogLevel]
```

Comment:
We should also check that `elClientsParticipantLogLevels` has our elClientType, right? That would be another bug in Kurtosis